app/src/main/res/layout/fragment_signup.xml

```diff
@@ -71,6 +71,10 @@
                 android:inputType="textPassword"
                 android:drawableStart="@drawable/ic_lock_black_24dp" />
         </android.support.design.widget.TextInputLayout>
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:hint="@string/password_constraints"/>
```

Comment:
This textview is not required IMO

Reply:
I think at the time of signup we should tell user about the constraints.